ee/tabby-webserver/src/cron/scheduler.rs

```diff
@@ -86,3 +39,61 @@ pub async fn scheduler_job(
 
     Ok(job)
 }
+
+async fn run_scheduler_now(
+    job: Arc<dyn JobService>,
+    worker: Arc<dyn WorkerService>,
+    local_port: u16,
+) -> Result<()> {
+    info!("Running scheduler job...");
+    let exe = std::env::current_exe()?;
+    let job_id = job.create_job_run("scheduler".to_owned()).await?;
+
+    let mut child = tokio::process::Command::new(exe)
+        .arg("scheduler")
+        .arg("--now")
+        .arg("--url")
+        .arg(format!("localhost:{local_port}"))
+        .arg("--token")
+        .arg(worker.read_registration_token().await?)
+        .stdout(Stdio::piped())
+        .stderr(Stdio::piped())
+        .spawn()?;
+
+    {
+        // Pipe stdout
+        let job = job.clone();
+        let job_id = job_id.clone();
+        let stdout = child.stdout.take().context("Failed to acquire stdout")?;
+        tokio::spawn(async move {
+            let stdout = tokio::io::BufReader::new(stdout);
+            let mut stdout = stdout.lines();
+            while let Ok(Some(line)) = stdout.next_line().await {
+                println!("{line}");
+                let _ = job.update_job_stdout(&job_id, line + "\n").await;
+            }
+        });
+    }
+
+    {
+        // Pipe stderr
+        let stderr = child.stderr.take().context("Failed to acquire stderr")?;
+        let job = job.clone();
+        let job_id = job_id.clone();
+        tokio::spawn(async move {
```

Comment:
very similar to the previous block, would it make sense to make this a function?

Reply:
not really something can be easily done:
1. stderr / stdout have different type.
2. they use eprintln / println differently.
3. they use update_job_stdout / update_job_stderr differently.